tensorboard/util/test_util.py

```diff
@@ -157,8 +158,12 @@ def _impl(reason=None):
 
   return _impl
 
-# TODO(#1996): Better detect TF2.0.
-_is_tf2 =  tf.__version__.startswith('2.')
+# Logic from `tensorflow.python.tf2`, assuming that `enable_v2_behavior`
```

Comment:
Can we just use `tensorflow.python.tf2.enabled()` Per https://github.com/tensorflow/tensorboard/issues/1996#issuecomment-490647019